pkg/apis/core/validation/validation.go

```diff
@@ -3165,6 +3167,17 @@ func validateContainerCommon(ctr *core.Container, volumes map[string]core.Volume
 		allErrs = append(allErrs, field.Required(path.Child("image"), ""))
 	}
 
+	// Adding API validation for images using the dockerref vendored package
+	if len(ctr.Image) > 0 {
+		// Whitespace in image name seems to be a valid case for our tests but docker says it isn't.
+		// We will ignore the check if we detect padded whitespace for now.
```

Comment:
These comments need to be updated as well.

Reply:
I believe this comment is correct.  

```
		// Whitespace in image name seems to be a valid case for our tests but docker says it isn't.
		// We will ignore the check if we detect padded whitespace for now.
		if len(ctr.Image) == len(strings.TrimSpace(ctr.Image)) {
			_, _, _, err := parsers.ParseImageName(ctr.Image)
			if err != nil {
				allErrs = append(allErrs, field.Invalid(path.Child("image"), ctr.Image, err.Error()))
			}
		}
```

Reply:
yes, it is. I probably confused it with another change. You can resolve this conversation.